.gitmodules

```diff
@@ -1,6 +1,8 @@
 [submodule "utils/dependencies"]
 	path = utils/dependencies
 	url = https://github.com/openebs/mayastor-dependencies.git
+	branch = release/2.0
```

Comment:
Can we use a tag later or maybe a plain commit hash(this will be less elegant but will be super-precise as to which dependency version we're depending upon).
Aside, I'm recalling a gem-of-a-GitHub action which used to do downstream tagging. 😉

Reply:
I think we're trying to pin to whatever is at the top of the release branch. That way, we're in sync with in-coming updates to the release code.